common-rest/src/main/java/org/uniprot/api/rest/service/StoreStreamerSearchService.java

```diff
@@ -102,6 +102,9 @@ public Stream<String> streamIdsForDownload(StreamRequest request) {
     }
 
     public QueryResult<R> getByIds(IdsSearchRequest idsRequest) {
+        if (idsRequest.getSize() == null) {
```

Comment:
Can you please move this logic to BasicRequestConverter.createSearchIdsSolrRequest? this is the new place that we handle with logics related with IdsSearchRequest. Can you please add a Unit Test for it as well?